go/consensus/tendermint/tendermint.go

```diff
@@ -699,6 +700,43 @@ func (t *tendermintService) GetTransactions(ctx context.Context, height int64) (
 	return txs, nil
 }
 
+func (t *tendermintService) GetStatus(ctx context.Context) (*consensusAPI.Status, error) {
+	genDoc, err := t.GetGenesisDocument(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	genBlk, err := t.GetBlock(ctx, genDoc.Height)
+	if err != nil {
+		return nil, err
+	}
+
+	latestBlk, err := t.GetBlock(ctx, consensusAPI.HeightLatest)
+	if err != nil {
+		return nil, err
+	}
+
+	addrs, err := t.GetAddresses()
+	if err != nil {
+		return nil, err
+	}
+
+	peers := make([]string, len(addrs))
```

Comment:
This creates a slice of strings, pre-populated with `len(addrs)` empty strings, that you are then appending to.

Reply:
Oops, I'll fix that, thanks!